src/flows/selectors/flowsSelectors.test.ts

```diff
@@ -0,0 +1,54 @@
+import {AppState} from 'src/types'
+import {selectShouldShowNotebooks} from 'src/flows/selectors/flowsSelectors'
+
+jest.mock('src/organizations/selectors', () => ({
+  selectOrgCreationDate: jest.fn(),
+}))
+
+import {selectOrgCreationDate} from 'src/organizations/selectors'
+
+describe('selecting whether notebooks should be shown', () => {
+  let fakeState
+  beforeEach(() => {
+    fakeState = {
+      resources: {
+        notebooks: {
+          notebooks: [],
+        },
+      },
+    } as AppState
+  })
+  it('does not show notebooks when the org creation date is after the IOx cutoff date', () => {
```

Comment:
Should we add a fake notebook in here for this "after" test - or break it into two tests? Unless `fakeState` has notebooks in it, we can't tell if the `false` state is due to the absence of notebooks or the cutoff date.

Reply:
sure, I can add in another case